source/people/aaswenson.rst

```diff
@@ -0,0 +1,29 @@
+Alexander Swenson
+=============
+
+Contact Information
+-------------------
+| `E-mail <aaswenson@wisc.edu>`_
+
+----
```

Comment:
I was just more so wondering why it was there since it is unnecessary. `---` usually indicates a header, so just having it there is unnecessary.